app/react/V2/Routes/Settings/Translations/components/FormInput.tsx

```diff
@@ -1,38 +0,0 @@
-import React from 'react';
-import { UseFormGetFieldState, UseFormRegister, UseFormSetValue } from 'react-hook-form';
-import { Translate } from 'app/I18N';
-import { InputField } from 'V2/Components/Forms';
-
-type fromPropsType = {
-  register: UseFormRegister<any>;
-  setValue: UseFormSetValue<any>;
-  getFieldState: UseFormGetFieldState<any>;
-  submitting: boolean;
-};
-
-const FormInput = (data: any, formProps: fromPropsType) => {
-  const { register, setValue, submitting, getFieldState } = formProps;
-  const { error } = getFieldState(data.cell.getValue());
-  const hasErrors = Boolean(error);
-  const reset = () => setValue(data.cell.getValue(), '', { shouldDirty: true });
-  const { onChange, onBlur, name, ref } = register(data.cell.getValue(), { required: true });
-
-  return (
-    <div>
-      <InputField
-        id={data.cell.getValue()}
-        label={data.cell.row.original.language}
-        hideLabel
-        disabled={submitting}
-        clearFieldAction={reset}
-        errorMessage={hasErrors ? <Translate>This field is required</Translate> : ''}
```

Comment:
By replacing this with the standard form input the `This field is required` message is not longer shown. Not a big deal but the error state for the new field is not that obvious, the border is not as red when it's not focused. In a long list an error might go unnoticed without this message.